front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -37,18 +40,25 @@ const defaultTranscriptConfigurationState = {
   isGongConnected: false,
   assistantSelected: null,
   isActive: false,
+  dataSource: null,
 };
 
 export const getServerSideProps = withDefaultUserAuthRequirements<{
   owner: WorkspaceType;
   subscription: SubscriptionType;
   gaTrackingId: string;
   dustClientFacingUrl: string;
+  dataSources: DataSourceType[];
 }>(async (_context, auth) => {
   const owner = auth.workspace();
   const subscription = auth.subscription();
   const user = auth.user();
 
+  const allDataSources = await getDataSources(auth, { includeEditedBy: true });
```

Comment:
cc @flvndvd do we have something we can start using right now here that will make our life easier as we roll out vaults?

Reply:
You can do something like this:
```
const globalVault = await VaultResource.fetchWorkspaceGlobalVault(auth);

const dataSourceViews = await DataSourceViewResource.listByVault(auth, globalVault);
```

And then on select pass the `dataSourceView.name` attributes, which maps to the data source name/sId.

Reply:
Ok for now - will move it to personal datasources whenever they are ready